api/scripts/modulix/get-modules-assets-debt.js

```diff
@@ -0,0 +1,228 @@
+import moduleDatasource from '../../src/devcomp/infrastructure/datasources/learning-content/module-datasource.js';
+
+const modules = await moduleDatasource.list();
```

Comment:
Je trouve qu'en terme de lisibilité, il faudrait plutôt wrapper ça dans une fonction `main` nn ? On a du mal à savoir où ça commence et où ça se termine 😆 

Quelque chose comme ça : 

```suggestion
async main() {
  const modules = await moduleDatasource.list();
  const allElements = getElements(modules);
  
  countAndDisplayVideosData();
  countAndDisplayImagesData();
  countAndDisplayIframesData();
}

await main();

// Les autres fonctions privées plus bas...
```